src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -231,5 +268,9 @@ export default compose(
             selector: policySelector,
             initialValue: {},
         },
+        transactionViolations: {
+            key: ONYXKEYS.TRANSACTION_VIOLATIONS,
```

Comment:
Too bad this file is still `.js` and not `.tsx` or this mistake would've been caught earlier. The correct key is `ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS`, not `ONYXKEYS.TRANSACTION_VIOLATIONS`